<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>550</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton {
	color: rgb(0, 0, 0);
	background: transparent;
    border-radius: 10px;
    padding: 10px;
    border: 1px solid black;
}

QPushButton:hover {
	background-color: rgb(206, 206, 206);
}

QPushButton:pressed {
    background-color: rgb(80, 80, 80);
};

background-color: rgb(214, 214, 214);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="play_pause">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/recursos/1200px-Play_Pause_icon_2283501.svg.png</normaloff>images/recursos/1200px-Play_Pause_icon_2283501.svg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="pts_azul">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>140</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid black;
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="subir_pts_azul">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/recursos/arriba.png</normaloff>images/recursos/arriba.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bajar_pts_azul">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/recursos/abajo.png</normaloff>images/recursos/abajo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="ctrl_pts_azul">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ctrl_tmp">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>20</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid black;
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="tiempo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>33</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none
</string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>45</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black</string>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tiempo_min">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>33</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="tiempo_seg">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>33</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none
</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="participante_azul">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>170</y>
      <width>230</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid black;
background-color: rgb(0, 0, 255);
border-radius: 8px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Participante Azul</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="participante_rojo">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>170</y>
      <width>230</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid black;
border-radius: 8px;
color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Participante Rojo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="pts_rojo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>140</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid black;
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="subir_pts_rojo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/recursos/arriba.png</normaloff>images/recursos/arriba.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bajar_pts_rojo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/recursos/abajo.png</normaloff>images/recursos/abajo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="ctrl_pts_rojo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="configuracion">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>10</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="pausado">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>Pausado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="dar_pts">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>250</y>
      <width>55</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border: 3px solid black;
	border-radius: 8px;
	background-color: rgb(0, 0, 0);
}

QPushButton { color: rgb(255, 0, 0); border: 2px solid rgb(255, 0, 0); }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="un_pto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dos_pts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>35</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cuatro_pts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>93</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cerrar_dar_pts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>135</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="tm_azul">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>435</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/recursos/Sin título.png</normaloff>
      <activeon>images/recursos/images.png</activeon>images/recursos/Sin título.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="tm_rojo">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>435</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/recursos/Sin título.png</normaloff>images/recursos/Sin título.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="amarilla_menos_azul">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>490</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="amarilla_mas_azul">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="amarilla_menos_rojo">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>490</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="amarilla_mas_rojo">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="amarillas_azul">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="icon_amariila1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>502</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
border: 2px solid black;
border-radius: 7px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="amarillas_rojo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="icon_amariila2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>502</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
border: 2px solid black;
border-radius: 7px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="azul_tm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>435</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string>2:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rojo_tm">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>435</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string>2:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
